x/auth/client/testutil/suite.go

```diff
@@ -588,7 +589,10 @@ func (s *IntegrationTestSuite) TestCLIQueryTxsCmdByEvents() {
 	s.Require().NoError(err)
 	var txRes sdk.TxResponse
 	s.Require().NoError(val.ClientCtx.Codec.UnmarshalJSON(out.Bytes(), &txRes))
-	s.Require().NoError(s.network.WaitForNextBlock())
+	height, err := s.network.LatestHeight()
+	s.Require().NoError(err)
+	_, err = s.network.WaitForHeight(height + 2) // this test fails unless we wait a couple blocks for some reason?
```

Comment:
really odd side effect form this PR - had to make this test wait a couple blocks, else it fails...